tests/devices/experimental/test_default_qubit_2.py

```diff
@@ -1278,6 +1278,36 @@ def test_global_seed(self, measurements, max_workers):
 
         assert all(np.all(res1 == res2) for res1, res2 in zip(result1, result2))
 
+    def test_global_seed_no_device_seed_by_default(self):
+        """Test that the global numpy seed initializes the rng if device seed is none."""
+        np.random.seed(42)
+        dev = DefaultQubit2()
+        first_num = dev._rng.random()  # pylint: disable=protected-access
+
+        np.random.seed(42)
+        dev2 = DefaultQubit2()
+        second_num = dev2._rng.random()  # pylint: disable=protected-access
+
+        assert qml.math.allclose(first_num, second_num)
+
+        np.random.seed(42)
+        dev2 = DefaultQubit2(seed="global")
+        third_num = dev2._rng.random()  # pylint: disable=protected-access
+
+        assert qml.math.allclose(third_num, first_num)
+
+    def test_None_seed_not_using_global_rng(self):
+        """Test that if the seed is None, it is uncorrelated with the global rng."""
+        np.random.seed(42)
+        dev = DefaultQubit2(seed=None)
+        first_num = dev._rng.random()  # pylint: disable=protected-access
+
+        np.random.seed(42)
+        dev2 = DefaultQubit2(seed=None)
+        second_num = dev2._rng.random()  # pylint: disable=protected-access
+
+        assert not qml.math.allclose(first_num, second_num)
+
```

Comment:
We could also add a test for PRNGs as input.
```suggestion

def test_rng_as_seed(self):
    """Test that a PRNG can be passed as a seed."""
    np.random.default_rng(42)
    first_num = rng.random()

    rng = np.random.default_rng(42)
    dev = DefaultQubit2(seed=rng)
    second_num = dev._rng.random()  # pylint: disable=protected-access

    assert qml.math.allclose(first_num, second_num)
```